documentation.md

```diff
@@ -1,13 +1,22 @@
 # Documentation
 
-We need to find a photographer to take pictures during the conference and a videographer so we could have the conference talks recorded. 
-
-Pictures should be uploaded during the events.
-
-The video should be uploaded after the event (preferably, within one week after PyCon). The video should be marked private. The public release of the video will be at the discretion of the board with this consideration: if the video is released too early, it lowers the value of PyCon tickets as people will simply wait for the videos.
```

Comment:
Out of curiosity why were these paragraphs deleted? They sound reasonable to me.